modules/jt-examples-of-common-use-cases.adoc

```diff
@@ -0,0 +1,169 @@
+:_content-type: CONCEPT
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-examples-of-common-use-cases_{context}"]
+= Examples of common use cases
+
+Both Jenkins and Tekton offer capabilities for common CI/CD use cases, such as: 
+
+* Compiling, building images, and deploying images
+* Extending the core capabilities by using plugins
+* Reusing shareable libraries and custom scripts
+
+This section briefly describes the comparable workflows for some common use cases in Jenkins and Tekton.
+
+== Running a maven pipeline in Jenkins and Tekton
+
+Maven is used extensively in the Jenkins workflows. You can map your existing workflow to Tekton by using the following examples:
```

Comment:
Not that I think passive voice is a big deal here, but if you did want to avoid it, you could rephrase it to something like:

- `Maven has an extensive role int eh Jenkins workflow.` 
- `The Jenkins workflow extensively incorporates Maven.` 
- `The Jenkins workflow includes/incorporates/utilizes/uses Maven.`

Reply:
I'm a little lost about the connection between these two sentences. The title does state `Running a maven pipeline in Jenkins and Tekton` so I expect to learn about all these topics. 

However, the first sentence tells me maven is used extensively in the Jenkins workflow. Then the next sentence tells me to map my workflow to Tekton, and use the examples to map my workflow. 

So, am I focusing my efforts to map my workflow to Tekton or learn about maven in Jenkins? Or both, in which case, how does maven in Jenkins workflow connect to Tekton workflows? 

Would this connection work?:

`Maven has an extensive role  in the Jenkins workflow. You can map your existing Jenkins workflow to Tekton by using the following examples:`

I'm not sure if the above suggestion is correct because I'm not sure if the workflows the users are mapping are Jenkins workflows. Probably not because the first sentence informs users that maven is extensive in Jenkins workflows. So if users don't know that, they're not familiar with Jenkins workflows....Basically, it seems like we jumped from Jenkins to Tekton without a segway.